zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/config/ElasticsearchProperties.java

```diff
@@ -22,6 +22,33 @@ public class ElasticsearchProperties {
   private IndexSettings defaultSettings = new IndexSettings();
   private Map<String, Integer> replicasByIndexName = new HashMap<>();
   private Map<String, Integer> shardsByIndexName = new HashMap<>();
+  private boolean retention;
```

Comment:
could retention be an inner class that encapsulates all retention configs? 
as in
```java
public static class RetentionConfiguration {
private boolean enabled = false;
    private String minimumAge = "30d";
    private String policyName;
}
```